ovn_cluster.sh

```diff
@@ -720,37 +830,37 @@ create_fake_vm() {
 create_fake_vm \$@
 
 EOF
-    chmod 0755 ${FAKENODE_MNT_DIR}/create_fake_vm.sh
+    chmod 0755 ${FAKENODE_MNT_DIR}/create_fake_vm.sh 
 
-    echo "Creating a fake VM in "${CHASSIS_NAMES[0]}" for logical port - sw0-port1"
-    ${RUNC_CMD} exec "${CHASSIS_NAMES[0]}" bash /data/create_fake_vm.sh sw0-port1 sw0p1 50:54:00:00:00:03 10.0.0.3 24 10.0.0.1 1000::3/64 1000::a
-    echo "Creating a fake VM in "${CHASSIS_NAMES[1]}" for logical port - sw1-port1"
-    ${RUNC_CMD} exec "${CHASSIS_NAMES[1]}" bash /data/create_fake_vm.sh sw1-port1 sw1p1 40:54:00:00:00:03 20.0.0.3 24 20.0.0.1 2000::3/64 2000::a
+    echo "Creating a fake VM in "${CHASSIS_NAMES[$((az-1))]}" for logical port - sw0$az-port1"
+    ${RUNC_CMD} exec "${CHASSIS_NAMES[$((az-1))]}" bash /data/create_fake_vm.sh sw0$az-port1 sw0${az}p1 50:5$az:00:00:00:03 1$az.0.0.3 24 1$az.0.0.1 100$az::3/64 100$az::a
+    echo "Creating a fake VM in "${CHASSIS_NAMES[$((az+CENTRAL_COUNT-1))]}" for logical port - sw1$az-port1"
```

Comment:
This doesn't work unless CHASSIS_COUNT >= 2 * CENTRAL_COUNT.  For example the following fails because CHASSIS_COUNT is 2 by default:
```
CREATE_FAKE_VMS=yes CENTRAL_COUNT=2 ./ovn_cluster.sh start
...
+ podman exec '' bash /data/create_fake_vm.sh sw11-port1 sw11p1 40:51:00:00:00:03 21.0.0.3 24 21.0.0.1 2001::3/64 2001::a
Error: name or ID cannot be empty
```
I think we need a check in the "start" handler to ensure that CHASSIS_COUNT >= 2 * CENTRAL_COUNT, wdyt?

Reply:
@dceara ack, I will fix it